mtp_noms_ops/apps/security/urls.py

```diff
@@ -386,8 +378,13 @@ def dashboard_view(request):
         name='resolve_check',
     ),
     url(
-        r'^security/checks/(?P<check_id>\d+)/assignment/(?P<list>\w+)?/(?P<current_page>\d+)?/?$',
-        fiu_security_test(views.CheckAssignView.as_view()),
+        r'^security/checks/(?P<check_id>\d+)/assignment/list/(?P<page>\d+)/$',
```

Comment:
Why is this refactor necessary for the design system upgrade?

Reply:
because i couldn't load the page locally anymore so i couldn't test the pages properly. the old url template gives you urls like:
`/en-gb/security/checks/502/assignment//`
but then when i click, i get a 404 saying this url does not match any pattern:
`/en-gb/security/checks/502/assignment/`
note the lack of double trailing `/`. for some reason, something locally is pre-sanitising the url before it hits the django router.

if you like i can pull this out into a separate pr, but i needed to change it to be able to test the button and what's displayed after it's clicked